spec/support/shared_examples/name_with_language.rb

```diff
@@ -0,0 +1,14 @@
+shared_examples 'name_with_language' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.